### PR TITLE
Document proofreads

### PR DESCRIPTION
**Pull request by Michael-Scott-PC**
I read Strapi's documentation, top-to-bottom, a few weeks ago and marked up any typos, grammatical & semantic errors. The documents have changed a decent amount since then, but these are the errors I found and fixed that are still relevant.

#### Description of what you did:
Documentation proofreading. Ready for merge.

My PR is a:
 💥 Breaking change
 🐛 Bug fix
[X] 💅 Enhancement
 🚀 New feature

Main update on the:
 Admin
[X] Documentation
 Framework
 Plugin

Manual testing done on the following databases:
[X] Not applicable
 MongoDB
 MySQL
 Postgres
 SQLite

### CODE REVIEW

**Comment by alexandrebodin** [comment]
Hi @Michael-Scott-PC are you willing to update your PR to fix the conflicts ?

**Comment by Michael-Scott-PC** [comment]
Hi @alexandrebodin sorry about that, not sure how I missed those.  Before I edited the files, I ran git remote add upstream https://github.com/strapi/strapi.git and then ran git fetch upstream, git checkout master, & git merge upstream/master.  I thought I had the most recent files but it looks like I might have still been editing old files, specifically the API endpoint file looks drastically different than what I edited.  A lot of the things I marked up are still in the docs (I just checked) but maybe I should re-run my commands and do this all over?

**Comment by alexandrebodin** [comment]
Looks like your PR is now up to date. I'll leave it to @lauriejim to review ;)

File: docs/3.0.0-beta.x/concepts/configurations.md [hunk 1]
```diff
@@ -480,15 +480,15 @@ As an example using this configuration with Nginx your server would respond to `
 
 ## Dynamic configurations
 
-For security reasons, sometimes it's better to set variables through the server environment. It's also useful to push dynamic values into configurations files. To enable this feature into JSON files, Strapi embraces a JSON-file interpreter into its core to allow dynamic values in the JSON configuration files.
+For security reasons, sometimes it's better to set variables through the server environment. It's also useful to push dynamics values into configuration files. To enable this feature in JSON files, Strapi embraces a JSON-file interpreter into its core to allow dynamic values in the JSON configuration files.
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/concepts/configurations.md]: I think it should still be `dynamic values` not `dynamics values`

File: docs/3.0.0-beta.x/concepts/controllers.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ In this example, any time a web browser is pointed to the `/hello` URL on your a
 
 ### Where are the controllers defined?
 
-The controllers are defined in each `./api/**/controllers/` folders. Every JavaScript file put in these folders will be loaded as a controller. They are also available through the `strapi.controllers` and `strapi.api.**.controllers` global variables. By convention, controllers' names should be Pascal-cased, so that every word in the file (include the first one) is capitalized `User.js`, `LegalEntity.js`.
+The controllers are defined in each `./api/**/controllers/` folder. Every JavaScript file put in these folders will be loaded as a controller. They are also available through the `strapi.controllers` and `strapi.api.**.controllers` global variables. By convention, controllers' names should be Pascal-cased, so that every word in the file (including the first one) is capitalized `User.js`, `LegalEntity.js`.
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/concepts/controllers.md]: Can you remove the Pascal-cased convention (it is outdated)

**Comment by alexandrebodin** [comment]
Hi @Michael-Scott-PC what you just push has a lot of conflicts :/. Can you fix this ? also please check the DCO check ;)

**Comment by Michael-Scott-PC** [comment]
I apologize, I've only ever used git/github working on my own.  I made one pull request with you guys a while back but I remember it being much simpler 😅. After I resolved the conflicts the first time, I ran the commands the DCO told me to sign off since I failed to the first time and I'm guessing that reset everything. I remember the DCO being a simple checkbox the first time I did a pull request. I see that I can also sign off via command line so I'll take care to do that when I push from my local repo.

**Pull request merged**